toolbox/+otp/RHS.m

```diff
@@ -1,36 +1,304 @@
+        %
+        % See Also
+        % --------
+        % JacobianMatrix
+        % JacobianFunction
         Jacobian
+
+        % The sparsity pattern of the Jacobian matrix.
+        %
+        % This follows the same specifications as in `odeset <https://www.mathworks.com/help/matlab/ref/odeset.html>`_.
         JPattern
+
+        % The mass matrix $M(t, y)$.
+        %
+        % This follows the same specifications as in `odeset <https://www.mathworks.com/help/matlab/ref/odeset.html>`_.
+        %
+        % See Also
+        % --------
+        % MassMatrix
+        % MassFunction
         Mass
+
+        % Whether the mass matrix is singular, i.e., the problem is a differential-algebraic equation.
+        %
+        % This follows the same specifications as in `odeset <https://www.mathworks.com/help/matlab/ref/odeset.html>`_.
         MassSingular
+
+        % State dependence of the mass matrix.
+        %
+        % This follows the same specifications as in `odeset <https://www.mathworks.com/help/matlab/ref/odeset.html>`_.
         MStateDependence
+
+        % The sparsity pattern of $\frac{\partial}{\partial y} M(t, y) v$.
+        %
+        % This follows the same specifications as in `odeset <https://www.mathworks.com/help/matlab/ref/odeset.html>`_.
         MvPattern
+
+        % A vector of solution components which should be nonnegative.
+        %
+        % This follows the same specifications as in `odeset <https://www.mathworks.com/help/matlab/ref/odeset.html>`_.
         NonNegative
+
+        % Whether $f$ can be evaluated at multiple states.
+        %
+        % This follows the same specifications as in `odeset <https://www.mathworks.com/help/matlab/ref/odeset.html>`_.
+        %
+        % Warning
+        % -------
+        % In Octave, this is always unset because it is not used by built in solvers and causes an error for ``ode15s``.
         Vectorized
+
+        % A function which detects events and determines whether to terminate the integration.
+        %
+        % This follows the same specifications as in `odeset <https://www.mathworks.com/help/matlab/ref/odeset.html>`_.
+        Events
+
+        % Response to a terminal event occuring.
+        %
+        % If set, it is a function handle with the following signature:
+        %
+        % Parameters
+        % ----------
+        % sol : struct
+        %    The solution generated by an ODE solver once an event occurs.
+        % problem : Problem
+        %    The problem for which the event occured. This should not be modified by the function.
+        %
+        % Returns
+        % -------
+        % terminal : logical
+        %    ``true`` if the integration should stop and ``false`` if it can continue.
+        % newProblem : Problem
+        %    A new Problem to use after the event, possibly with the time span, initial conditions, or parameters
+        %    updated.
+        %
+        % Warning
```

Comment:
Is there a way to make this red/orange, or in a box? I feel like warnings should be highlighted in some way.

Reply:
Already is